src/css/components/icons.less

```diff
@@ -107,6 +107,12 @@
     background-size: contain;
   }
 
+  &.cpu {
+    background-image: url("img/icons/icon-mini-cpu.svg");
```

Comment:
I do not see it in https://github.com/criteo-forks/marathon-ui/tree/criteo/src/img/icons
Am I missing something? :)